docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.2'
 
 services:
     reverseproxy:
-        image: reverseproxy
+        build: docker-nginx
```

Comment:
This means it's no longer using our Nginx configuration (redirects and stuff).

Reply:
How do you mean? This means it builds the image using the docker-nginx subdirectory when doing `docker-compose up` or `docker-compose build`, and does not try to pull a nonexisting image on `docker-compose pull`.

Reply:
Could be that I missed something in `docker-nginx/Dockerfile`, I'll check that

Reply:
Nope, this is correct